<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>831</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>831</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
	background-color: rgb(85, 255, 255);
background-color:qlineargradient(spread:pad, x1:0.096, y1:0.158955, x2:0.893389, y2:0.813, stop:0 rgba(29, 162, 216, 255), stop:1 rgba(222, 243, 246, 255));}
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;color:white;</string>
    </property>
    <property name="text">
     <string>QUESTIONNAIRE </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;color:black;
font: 14pt &quot;MS Shell Dlg 2&quot;;color:black;</string>
    </property>
    <property name="text">
     <string>Please fill in the questionnaire with yes/no based on the questions </string>
    </property>
   </widget>
   <widget class="QPushButton" name="proceed">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>720</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color:#064273;
font: 16pt &quot;MS Shell Dlg 2&quot;;color:#def3f6;
</string>
    </property>
    <property name="text">
     <string>PROCEED</string>
    </property>
   </widget>
   <widget class="QLabel" name="errormsg">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>740</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="yes9">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>290</y>
      <width>62</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>YES</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">q1</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="no9">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>290</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>NO</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">q1</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="yes10">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>410</y>
      <width>62</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>YES</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">q2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="no10">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>410</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>NO</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">q2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="yes11">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>530</y>
      <width>62</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>YES</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">q3</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="no11">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>530</y>
      <width>62</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>NO</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">q3</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="yes12">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>660</y>
      <width>62</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>YES</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">q4</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="no12">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>660</y>
      <width>62</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>NO</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">q4</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="Q7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>9. Have you had dengue or typhoid in the 
last 1 year? </string>
    </property>
   </widget>
   <widget class="QLabel" name="Q8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>10. Have you taken any antibiotics/aspirin/alcohol/
steroids in the last 72 hours? </string>
    </property>
   </widget>
   <widget class="QLabel" name="Q11">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>551</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>11. Have you been pregnant or had an abortion in 
the last six months?</string>
    </property>
   </widget>
   <widget class="QLabel" name="Q12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>650</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>12. Have you received the COVID 19 Vaccine?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="q1"/>
  <buttongroup name="q2"/>
  <buttongroup name="q3"/>
  <buttongroup name="q4"/>
 </buttongroups>
</ui>
